torch/_dynamo/variables/torch.py

```diff
@@ -220,77 +237,11 @@ def call_function(
             return TorchFunctionDisableVariable.create(tx)
 
 
-class TorchVariable(VariableTracker):
-    """Points to a module or method in torch.*"""
-
-    def __init__(self, value, **kwargs):
-        super().__init__(**kwargs)
-        if (
-            isinstance(value, collections.abc.Hashable)
-            and value in tensor_dunder_fns_remap
-        ):
-            value = tensor_dunder_fns_remap[value]
-
-        self.value = value
-
-        assert not isinstance(
-            value, (torch.dtype, torch.device)
-        ), "should use ConstantVariable"
```

Comment:
This assert seems to have been lost

Reply:
Right, I missed it during resolving conflicts, I'll add it back.

Reply:
Resolved!